lib/Room.php

```diff
@@ -394,16 +391,7 @@ public function setDescription(string $description): void {
 		$this->description = $description;
 	}
 
-	/**
-	 * @deprecated Use ParticipantService::getGuestCount() instead
-	 * @return int
-	 */
-	public function getActiveGuests(): int {
```

Comment:
This is a value on the model, please keep it injected and also keep the getter + setter to modify the value on the model (only the DB update part is basically moved to the RoomService